src/pudl/metadata/classes.py

```diff
@@ -1194,6 +1194,7 @@ class Resource(Base):
         "ferc714",
         "glue",
         "outputs",
+        "output_views",
```

Comment:
This is no longer necessary now that we can filter view metadata out using the `include_in_database` attribute.